uber/templates/registration/view_promo_codes.html

```diff
@@ -0,0 +1,230 @@
+{% extends "base-admin.html" %}
+{% block title %}View Promo Codes{% endblock %}
+{% block content %}
+<style>
+.decent > th,td{
+    padding: 1em;
+}
+.decent-rows:hover{
+    background-color: #f5f5f5;
+}
+.delete{
+    font-size: 36px;
+}
+
+.delete:hover{
+    color: #FF0011;
+}
+</style>
+<h2> View Promo Codes </h2>
+
+<table>
+<tr class="decent">
+    <th>
+        Delete?
+    </th>
+    <th>
+        Code
+    </th>
+    <th>
+        Expiration Date
+    </th>
+    <th>
+        Uses
+    </th>
+    <th>
+        Ticket Price
+    </th>
+    <th>
+        Expired
+    </th>
+    <th>
+        Send Email?
+    </th>
+    <th>
+        Users
+    </th>
+</tr>
+{% for p in promo_codes %}
+    <tr class="decent decent-rows">
+        <td class="delete">
+            <a href="#" onclick="deleteCode('{{ p.id }}')"><i>X</i></a>
```

Comment:
This "X" is massive and ugly on the page. I have no idea how you made it that gigantic.

Additionally, the UX for this in general is underwhelming. There's no confirmation pop-up to make sure the admin wants this code deleted. The toastr message just says 'Success!' -- it doesn't tell you what you just _did_. And the code apparently sticks around until the toastr message expires, then the whole page refreshes, which means you are potentially interrupting the admin doing something else AND allowing the admin to try to delete the same code multiple times. All of these things are pretty bad.